src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -207,8 +208,8 @@ object DockerPlugin extends AutoPlugin {
    * @param exposedPorts
    * @return if ports are exposed the EXPOSE command
    */
-  private final def makeExposePorts(exposedPorts: Seq[Int]): Option[CmdLike] = {
-    if (exposedPorts.isEmpty) None else Some(Cmd("EXPOSE", exposedPorts mkString " "))
+  private final def makeExposePorts(exposedPorts: Seq[Int], exposedUdpPorts: Seq[Int]): Option[CmdLike] = {
```

Comment:
Do we care? I think most of the methods in this class could have a more limited scope.